Javascript_Assignment_1.html

```diff
@@ -0,0 +1,61 @@
+<html>
+<head>
+
+	<script type="text/javascript">
+
+	function occurenceArray(){	
```

Comment:
Your function name should appropriate. the 'occurenceArray' name does not clear about the functionality.